packages/react-events/src/Press.js

```diff
@@ -436,10 +436,32 @@ function getAbsoluteBoundingClientRect(
   let offsetY = 0;
 
   // Traverse through all offset nodes
-  while (node != null && node.nodeType !== Node.DOCUMENT_NODE) {
-    offsetX += (node: any).scrollLeft;
-    offsetY += (node: any).scrollTop;
-    node = node.parentNode;
+  while (node != null) {
+    const parent = node.parentNode;
+    const scrollTop = node.scrollTop;
```

Comment:
Since these causes the browser to synchronously reflow it could be worth moving this inline in the condition below so it's avoided when possible

Reply:
It's potentially used twice though, so accessing it twice is worse than once?

Reply:
Actually, I wasn't re-using the stored variable, now I am. As it is used on either path.